index.js

```diff
@@ -90,21 +103,25 @@ function createHarness(conf_) {
 	}
 	test._results = results;
 
-	test._tests = [];
+	/** @type {Test[]} */ test._tests = [];
 
+	/** @type {import('.').CreateStream} */
```

Comment:
FWIW, I personally have found it to be more common & readable to declare imports once each on top of the file the way you have with the `@typedef`s. It's not a requirement and I wouldn't likely want to block on it for this repo, but I like the consistency of approach & better readability of the `@type` comments.

Reply:
that seems reasonable, i can do that. was trying to minimize the comment weight in the filesize.